hertzbeat-alerter/src/main/java/org/apache/hertzbeat/alert/service/impl/AlertServiceImpl.java

```diff
@@ -190,7 +189,7 @@ public void addNewAlertReportFromCloud(String cloudServiceName, String alertRepo
             alert = AlertReport.builder()
                     .content("error do not has cloud service api")
                     .alertName("/api/alerts/report/" + cloudServiceName)
-                    .alertTime(new Date().getTime())
+                    .alertTime(System.currentTimeMillis())
```

Comment:
use 'Instant.now().getEpochSecond()'

Reply:
> use 'Instant.now().getEpochSecond()'

Ok, it has been modified according to your instructions